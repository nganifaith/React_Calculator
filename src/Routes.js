import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import Calculator from './components/Calculator';
import Home from './components/Home';
import Header from './components/Header';

const Routes = () => (
  <BrowserRouter>
    <Header />
    <Switch>
      <Route exact path="/" component={Home} />
      <Route path="/calculator" component={Calculator} />
    </Switch>
  </BrowserRouter>
);

export default Routes;
