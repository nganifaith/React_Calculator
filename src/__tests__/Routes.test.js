import renderer from 'react-test-renderer';
import Routes from '../Routes';

it('renders pages correctly', () => {
  const router = renderer.create(<Routes />).toJSON();
  expect(router).toMatchSnapshot();
});
