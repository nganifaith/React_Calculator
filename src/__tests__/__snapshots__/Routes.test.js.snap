// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders pages correctly 1`] = `
Array [
  <nav>
    <h1>
      Math Magicians
    </h1>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
           Calculator
        </a>
      </li>
      <li>
        <a
          href="/quotes"
          onClick={[Function]}
        >
           Quotes 
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="welcome"
  >
    <h1>
       Welcome to Our Page
    </h1>
    <p>
      Counting objects, inside and outside, longer and shorter, letter names, rhyming words, and more. Comparing numbers, names of shapes, consonant and vowel sounds, sight words, and more. Adding and subtracting, tens and ones, telling time, categories, nouns, verb tense, time order, and more. Place-value models, contractions, irregular plurals, plants and animals, historical figures, and more. Multiplying and dividing, bar graphs, pronouns, possessives, weather and climate, geography, and more
    </p>
    <p>
      Counting objects, inside and outside, longer and shorter, letter names, rhyming words, and more. Comparing numbers, names of shapes, consonant and vowel sounds, sight words, and more. Adding and subtracting, tens and ones, telling time, categories, nouns, verb tense, time order, and more. Place-value models, contractions, irregular plurals, plants and animals, historical figures, and more. Multiplying and dividing, bar graphs, pronouns, possessives, weather and climate, geography, and more
    </p>
  </div>,
]
`;
